django/forms/models.py

```diff
@@ -642,16 +665,34 @@ def save(self, commit=True):
         as necessary, and returns the list of instances.
         """
         if not commit:
-            self.saved_forms = []
-
-            def save_m2m():
-                for form in self.saved_forms:
-                    form.save_m2m()
-            self.save_m2m = save_m2m
+            self._update_save_m2m()
         return self.save_existing_objects(commit) + self.save_new_objects(commit)
 
     save.alters_data = True
 
+    def get_updated_models(self):
```

Comment:
This method doesn't look to be used, in this patch based on a quick `find-on-page`?